JsonClient/JsonClient/JsonClient.cs

```diff
@@ -53,38 +53,42 @@ public JsonClient()
             }
         }
 
-    //    public async Task<Comment> postComment(Client client, Comment comment){
-    //         //post Comment
-    //         try{
-    //             HttpResponseMessage response = await client.PostAsJsonAsync("comments", comment);
-    //             response.EnsureSuccessStatusCode();
+        public async Task<TModel> PostAsync(TModel model){
+             //post Comment
+             try{
+                 var requestURL = $"{typeof(TModel).Name.ToLower()}s";
```

Comment:
Can we refactor this line? We can create a private field which is initialized in the constructor instead of retrieve the model's name every time the method is called.